jetty-io/src/main/java/org/eclipse/jetty/io/LogarithmicArrayByteBufferPool.java

```diff
@@ -65,7 +65,7 @@ public LogarithmicArrayByteBufferPool(int minCapacity, int maxCapacity, int maxQ
      */
     public LogarithmicArrayByteBufferPool(int minCapacity, int maxCapacity, int maxQueueLength, long maxHeapMemory, long maxDirectMemory)
     {
-        this(minCapacity, maxCapacity, maxQueueLength, maxHeapMemory, maxDirectMemory, maxHeapMemory, maxDirectMemory);
+        this(minCapacity, maxCapacity, maxQueueLength, maxHeapMemory, maxDirectMemory, -2, -2);
```

Comment:
Do we need to document this change in defaults?

Reply:
I'm in favour of option 3.   Let's not change behaviour significantly in a dot release.  
So restore the behaviour here but document the configuration changes needed to get the no retained memory behaviour.